src/libcaf.h

```diff
@@ -296,6 +298,7 @@ void PREFIX (form_team) (int, caf_team_t *, int);
 void PREFIX (change_team) (caf_team_t *, int);
 void PREFIX (end_team) (caf_team_t *);
 void PREFIX (sync_team) (caf_team_t *, int);
+int PREFIX (team_number) (caf_team_t *);
```

Comment:
in `libcaf.h` I have removed the `#ifdef GCC_GE_7` around the `caf_team_t` type def. I have no idea if this is unsafe for any reason, but it successfully lets the 6.x branch compile and pass all tests. I'm guessing it's fine since the compiler calls library functions having to do with teams, but I could be wrong. Feedback much appreciated by those who know better.